db/videos_information.csv

```diff
@@ -0,0 +1,9 @@
+title,description,small_cover,large_cover,category
```

Comment:
If you put some csv's like this to the project, I would recommend `/lib` and add it loaded paths.